qubes/ext/audio.py

```diff
@@ -0,0 +1,64 @@
+#
+# The Qubes OS Project, https://www.qubes-os.org/
+#
+# Copyright (C) 2019 Frédéric Pierret <frederic.pierret@qubes-os.org>
+#
+# This library is free software; you can redistribute it and/or
+# modify it under the terms of the GNU Lesser General Public
+# License as published by the Free Software Foundation; either
+# version 2.1 of the License, or (at your option) any later version.
+#
+# This library is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this library; if not, see <https://www.gnu.org/licenses/>.
+#
+
+import qubes.config
+import qubes.ext
+
+
+class AUDIO(qubes.ext.Extension):
+    # pylint: disable=too-few-public-methods
+    # property-del <=> property-reset-to-default
+    @qubes.ext.handler('property-del:audiovm')
+    def on_property_del(self, subject, event, name, oldvalue=None):
+        newvalue = getattr(subject, 'audiovm', None)
+        self.on_property_set(subject, event, name, newvalue, oldvalue)
+
+    @qubes.ext.handler('property-set:audiovm')
+    def on_property_set(self, subject, event, name, newvalue, oldvalue=None):
+        # pylint: disable=unused-argument,no-self-use
+
+        # Clean other 'audiovm-XXX' tags.
+        # gui-agent can connect to only one domain
```

Comment:
pulseaudio agent AKA module-vchan-sink

Reply:
Ok thank you. I pushed force with this correction.